app/controllers/order_items_controller.rb

```diff
@@ -0,0 +1,50 @@
+class OrderItemsController < ApplicationController
+  before_action :current_user
+
+  def create
+    my_order
```

Comment:
If you are calling methods from the `ApplicationController` or any other place, to make it easier on your teammates and other people who will be inheriting your code, put a comment that it is a method call and which controller it comes from if it is not in the current controller.
e.g.
```ruby
def create
  my_order # method call from Application Controller
```